app/templates/content/index.hbs

```diff
@@ -17,7 +17,7 @@
                                 {{#each authors as |author| ~}}
                                     {{#if author.bibliographic}}
                                         {{#if author.unregisteredContributor}}
-                                            <li>{{author.unregisteredContributor}}</li>
+                                            {{claim-user author=author}}
```

Comment:
this needs logic to prevent logged in people from seeing this view. They should see the deleted line above